buildSrc/src/main/groovy/library.java-conventions.gradle

```diff
@@ -12,6 +12,8 @@ compileJava {
     // when only a more recent JDK is available (for example, compile for Java 11 with Java 17 or 21).
     sourceCompatibility = '1.8'
     targetCompatibility = '1.8'
+    options.debug = true
```

Comment:
Do we want to leave this, or maybe make it conditional on Gradle debug options being set?

Reply:
This is required to enable debug functionality when running WRIMS GUI computes. Comments have been added to the java-conventions.gradle file.